test/js/bun/test/expect.test.ts

```diff
@@ -354,6 +354,33 @@ describe("expect()", () => {
     expect("bob").not.toInclude("alice");
   });
 
+  test("toIncludeRepeated()", () => {
```

Comment:
Should add test cases for:
* n = 0
* n = 0 with not
* value is empty string
* include is empty string
* both value and include are empty strings
* emoji strings